README.md

```diff
@@ -53,7 +55,17 @@ colors for then inner gauge are shaded by 25%.
 
 The list is automatically sorted so you don't need to do that in your config - but I recommend it anyways.
 
-## Example
+### min and max config
+
+There is 3 ways to config the minimum and maximum value of the gauge:
```

Comment:
Actually there are two ways to configure this and a default when you dont configure it at all.

Is suggest something like:
`The min and max values may be configured for both gauges at once or for each gauge individualy and default to 0 to 100.`
as for the color config.